project.clj

```diff
@@ -22,4 +22,5 @@
                  [org.clojure/clojurescript :scope "provided"]
                  [thheller/shadow-cljs :scope "provided"]
                  [pjstadig/humane-test-output "0.10.0"]]
-  :profiles {:dev [:parent/dev {:dependencies [[org.slf4j/slf4j-simple "1.7.30"]]}]})
+  :profiles {:dev [:parent/dev {:dependencies [[org.slf4j/slf4j-simple "1.7.30"]
+                                               [lein-parent "0.3.7"]]}]})
```

Comment:
Had to add this as a dev dependency otherwise the tests failed to find it (and lein doesn't load plugins dynamically)